packages/main/src/SelectMenu.ts

```diff
@@ -0,0 +1,275 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
+import { isPhone } from "@ui5/webcomponents-base/dist/Device.js";
+
+// Template
+import SelectMenuTemplate from "./generated/templates/SelectMenuTemplate.lit.js";
+
+// Styles
+import SelectMenuCss from "./generated/themes/SelectMenu.css.js";
+import ValueStateMessageCss from "./generated/themes/ValueStateMessage.css.js";
+import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverCommon.css.js";
+
+// Deps
+import ResponsivePopover from "./ResponsivePopover.js";
+import List from "./List.js";
+import Button from "./Button.js";
+
+// Types
+import type Select from "./Select.js";
+import type SelectMenuOption from "./SelectMenuOption.js";
+
+type SelectMenuOptionClick = {
+	option: SelectMenuOption,
+	optionIndex: number,
+};
+
+type SelectMenuChange = {
+	text: string,
+	selectedIndex: number,
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ *
+ * <h3>Usage</h3>
+ *
+ * For the <code>select-menu</code>
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/SelectMenu.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.main.SelectMenu
+ * @extends sap.ui.webc.base.UI5Element
+ * @tagname select-menu
```

Comment:
Missing since attribute

Reply:
ah yes, it's not added anywhere, we wanted to first align with everyone and then see for which version it would be ready for